package.json

```diff
@@ -119,6 +122,11 @@
     "js-sha3": "^0.8.0",
     "katex": "^0.12.0",
     "luxon": "^2.0.2",
+    "markdown-it": "12.2.0",
```

Comment:
We don't really. But it gets added automatically when one of the dependencies in deps.edn gets installed.

I thought it was fluree because last time I checked fluree added a bunch (from https://github.com/fluree/db/blob/main/src/deps.cljs), but these aren't there. So I don't which clojure dep does it, but one of them does.

Reply:
![wtf](https://media.giphy.com/media/Tf310Fi4pnQVYS6r4L/giphy.gif)